docs/docs/concepts/index.md

```diff
@@ -75,6 +75,7 @@ The LangGraph Platform comprises several components that work together to suppor
 - [Cron Jobs](./langgraph_server.md#cron-jobs): Cron jobs are a way to schedule tasks to run at specific times in your LangGraph application.
 - [Double Texting](./double_texting.md): Double texting is a common issue in LLM applications where users may send multiple messages before the graph has finished running. This guide explains how to handle double texting with LangGraph Deploy.
 - [Authentication & Access Control](./auth.md): Learn about options for authentication and access control when deploying the LangGraph Platform.
+- [MCP](./server-mcp.md): Expose your LangGraph agents as MCP tools using Streamable HTTP transport.
```

Comment:
For "THE" mcp "concept" i would expect it to cover both client and server - using mcp tools from a langgraph agent and exposing a langgraph deployment as an mcp tool

Reply:
This one is nested under LangGraph Server. Should I re-title as MCP Endpoint for now?

Reply:
I think so. Just for clarity when searching. People say the darndest things when they talk about mcp